frappe/public/scss/desk/kanban.scss

```diff
@@ -32,7 +25,8 @@
 		border-radius: var(--border-radius);
 		padding: var(--padding-md);
 		min-height: calc(100vh - 250px);
-		max-height: calc(100vh - var(--navbar-height) - var(--page-bottom-margin) - 80px);
+		background-color: var(--kanban-column-bg);
```

Comment:
--kanban-column-bg is slightly dark. Let's use --bg-color instead.

|  Before  |  After  |
|  -------  |  -----  |
|  <img width="846" alt="image" src="https://user-images.githubusercontent.com/30859809/178677837-e8fee994-b5c5-4875-8e21-e2a25eeb7754.png">  |  <img width="887" alt="image" src="https://user-images.githubusercontent.com/30859809/178677797-faff7ead-ddb4-4f1c-8b48-35273a8d8d96.png">  |